src/components/Policy/WizardSteps/ActionsStep.tsx

```diff
@@ -20,6 +21,7 @@ const ActionsStep = () => {
 
     return (
         <Form>
+            <Label>{Messages.wizardActions}</Label>
```

Comment:
I'm sure the linter will complain about this line:

```suggestion
            <Label>{ Messages.wizardActions }</Label>
```
You can check by running `yarn lint`